plugins/modules/snap.py

```diff
@@ -29,10 +30,12 @@
     name:
         description:
             - Name of the snaps to be installed.
-            - Any named snap accepted by the C(snap) command is valid.
+            - Any named snap accepted by the C(snap install) command is valid.
+              You can verify if a given snap is available on the Snap Store in advance by running: C(snap info <snap-name>).
```

Comment:
I tend to agree with @farshidtz .

Reply:
In case this sentence will be kept in some form, please note that it is not a string, but a one-element YAML dictionary with key `You can verify if a given snap is available on the Snap Store in advance by running` and value `C(snap info <snap-name>).` :)